File: app/src/main/java/xyz/sleekstats/completist/model/WatchCount.java
package xyz.sleekstats.completist.model;

public class WatchCount {

    private int watched;
    private int total;

    public WatchCount(int watched, int total) {
        this.watched = watched;
        this.total = total;
    }

    public int getWatched() {
        return watched;
    }

    public void setWatched(int watched) {
        this.watched = watched;
    }

    public int getTotal() {
        return total;
    }

    public void setTotal(int total) {
        this.total = total;
    }

    public int getWatchedPct() {
        if(this.total == 0) {
            return 0;
        }
        return (this.watched * 100)/this.total;
    }

    @Override
    public String toString() {
        return "Watched: " + this.watched + "/" + this.total + " (" + this.getWatchedPct() + "%)";
    }
}

File: app/src/main/java/xyz/sleekstats/completist/model/WatchCount.kt
package xyz.sleekstats.completist.model

class WatchCount(var watched: Int, var total: Int) {

    val watchedPct: Int
        get() = if (total == 0) {
            0
        } else watched * 100 / total

    override fun toString(): String {
        return "Watched: $watched/$total ($watchedPct%)"
    }

}